packages/react-image/src/components/Image/useImage.ts

```diff
@@ -0,0 +1,38 @@
+import { getStyleFromPropsAndOptions } from '@fluentui/react-theme-provider';
+import { ComposePreparedOptions } from '@fluentui/react-compose';
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// @ts-ignore Typings require esModuleInterop
```

Comment:
Would it be possible to fix the typings, or does this use a module structure where that's not possible? (I remember that being an issue with some structures but forget which ones specifically)

Reply:
```ts
import objectFitImages from 'object-fit-images';
```
```
Error:(3, 8) TS1259: Module '"office-ui-fabric-react/node_modules/@types/object-fit-images/index"' can only be default-imported using the 'esModuleInterop' flag
```

**vs.**

```ts
import * as objectFitImages from 'object-fit-images';
```
```
Error:(3, 34) TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.
```

---

- Option 1: enable `esModuleInterop` on this/all packages
- Option 2: fix typings (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/object-fit-images/index.d.ts), but I am familiar enough to do this. Can you please help me there?

Reply:
Hmm in this case it looks like the published version of `object-fit-images` uses `module.exports =`, which unfortunately means the typings do also need to use `export =`. I'm surprised TS complains about the `import *` in that case though...